back-end/API/tests/test_employe_jobOffer.py

```
@@ -0,0 +1,647 @@
+        "salary": '1000',
+        "offerDebut": "2021-12-12",
+        "active": True,
+        "approbationMessage": "Super offre!",
+        "employerId": 1,
+        "isApproved": True
+        },
+        "studyPrograms": [5, 6] ,
+        "scheduleIds": [1, 2]
+    }
+
+    response = client.put(f'/jobOffer/1', json=data)
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)